src/index.js

```diff
@@ -20,4 +21,4 @@ const styled = _styled(_styledComponent(_ComponentStyle(generateAlphabeticName))
 
 /* Export everything */
 export default styled
-export { css, keyframes, injectGlobal, ThemeProvider }
+export { css, keyframes, injectGlobal, ThemeProvider, ServerStyleSheet }
```

Comment:
@geelen you do realize this comment contradicts the one in the CHANGELOG.md comment right?  You are wanting to remove the capitalization in the export even though you want capitalization in the CHANGELOG?